mungegithub/features/repo-updates.go

```diff
@@ -132,9 +132,10 @@ func (o *RepoInfo) walkFunc(path string, info os.FileInfo, err error) error {
 		return err
 	}
 	path = filepath.Dir(path)
-	// Make the root explicitly / so its easy to distinguish. Nothing else is `/` anchored
+	// Make the root explicitly "" (empty string) for consistency with githubapi convention: nothing is `/` anchored
+	// so the base dir of the project is "" and all other files do not start with "/"
 	if path == "." {
-		path = "/"
+		path = ""
```

Comment:
We could wrap all the calls to filePath in a `canonicalizePath` function which always gives out a path the way we want it. It I think that would be preferable to `if..else..` cases like this.